src/test/java/com/vinyl/vinylstore/LoginTest.java

```diff
@@ -73,7 +73,7 @@ public void loginWithValidEmailAndPasswordWhenTokenDoesNotExist() {
 		ResponseEntity<TokenDTO> tdo = trt.postForEntity("/users/login", request, TokenDTO.class);
 
 		Assertions.assertThat(tdo.getStatusCode()).isEqualTo(HttpStatus.OK);
-		Assertions.assertThat(tdo.getBody().getHash());
+		Assertions.assertThat(tokenRepository.findByUserId(user.getId()));
```

Comment:
Assertions.assertThat([The Object]).isNotNull();

Reply:
done